lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -13,12 +13,16 @@ variables:
   DEFAULT_IMAGE: "jangrewe/gitlab-ci-android"
   DEPLOY_DEBUG_APK_SLACK_MESSAGE: "Hello Team! Here is the latest debug APK from branch `${CI_COMMIT_REF_NAME}`. It was triggered by: `${CI_PIPELINE_SOURCE}`."
   DEPLOY_DEBUG_APK_PATH: app/build/outputs/apk
+  DEPLOY_DEBUG_AAR_PATH: app/build/outputs/aar
   DEPLOY_DEBUG_APK_NAMES: yourdebugapkname
   DEPLOY_RELEASE_APK_SLACK_MESSAGE: "Hello Team! Here is the latest release APK triggered by tag: `${CI_COMMIT_TAG}`"
   DEPLOY_RELEASE_APK_PATH: app/build/outputs/apk
+  DEPLOY_RELEASE_AAR_PATH: app/build/outputs/aar
   DEPLOY_RELEASE_APK_NAMES: yourreleaseapkname
   APK_SLACK_CHANNEL_ACCESS_TOKEN: ""
   APK_SLACK_CHANNEL_ID: ""
+  ARTIFACTORY_SNAPSHOT_URL: ""
+  ARTIFACTORY_RELEASE_URL: ""
```

Comment:
With the maven-plugin these can be used for non-Artifactory servers such as Nexus. Can you please change these to be more generic such as `ARTIFACT_SNAPSHOT_URL` and `ARTIFACT_RELEASE_URL`? Also change `artifactoryUsername`, `artifactoryPassword`, and `artifactoryUrl`  to be more generic like `artifactUsername`, `artifactPassword`, and `artifactUrl`?